mgmtsystem/openstack.py

```diff
@@ -39,12 +39,10 @@ def _request_timeout_handler(self, url, method, retry_count=0, **kwargs):
         return HTTPClient.request(self, url, method, **kwargs)
     except Timeout:
         if retry_count >= 3:
-            self.logger.error('nova request timed out after {} retries'.format(retry_count))
             raise
         else:
             # feed back into the replaced method that supports retry_count
             retry_count += 1
-            self.logger.info('nova request timed out; retry {}'.format(retry_count))
             return self.request(url, method, retry_count=retry_count, **kwargs)
 
 
```

Comment:
So we _could_

```self._api.client._cfme_logger = logger```

And then modify the lines above to do ```self._cfme_logger```

Would that work? Basically monkeypatch in the logger?